PerformanceSummaryToCsv/TaskSummary.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace PerformanceSummaryToCsv
 {
-    public record TaskSummary(string Name, uint DurationMS, uint Invocations)
+    public record TaskSummary(string Name, dynamic DurationMS, int Invocations)
```

Comment:
seeing `dynamic` makes me nervous. Why are these changes necessary?

Reply:
The uint to dynamic change was because I didn't want to change all of your versions to doubles, the ETW traces record them with fractional ms, and I was interested in keeping that. This let us have both.

I'm not sure how to capture the number of items grouped with a GroupBy, and it seemed like less critical information, so I replaced it with -1 always. I think I can get the count data, though, so I can undo the uint -> int change afterwards if you'd like.

Reply:
it looks like I'm not currently emitting the count so just drop it from the TaskSummary record.

I'm fine with switching the measurements to doubles.